dsc/tests/dsc_export.tests.ps1

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'resource export tests' {
+    
+    It 'Export can be called on individual resource' {
+
+        $out = dsc resource export -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $config_with_process_list = $out | ConvertFrom-Json
+        $config_with_process_list.resources.count | Should -BeGreaterThan 1
```

Comment:
do some validation that this is a config doc

Reply:
Updated.